subt/main.py

```diff
@@ -831,7 +831,7 @@ def go_to_entrance(self):
         trace.add_line_to((0.5, entrance_offset, self.height_above_ground))  # 0.5m inside, towards the desired wall.
         trace.reverse()
         is_trace3d = self.height_above_ground > 0.0  # well, maybe there should be more obvious definition of ground/aerial vehicle
-        self.follow_trace(trace, timeout=timedelta(seconds=30), max_target_distance=2.5, safety_limit=0.2, is_trace3d=is_trace3d)
+        self.follow_trace(trace, timeout=timedelta(seconds=30), max_target_distance=2.5, safety_limit=None, is_trace3d=is_trace3d)
```

Comment:
I am not sure about this one ... I would maybe change it for drones only??

Reply:
Good point. Perhaps. So conditioned on `is_trace3d`?

Reply:
yeah, good idea - I will change it once it takes off :)